salvo/src/lib/benchmark/test_common.py

```diff
@@ -0,0 +1,81 @@
+"""
```

Comment:
Can we come up with a better (more specific) name for this library? If we name it `test_common`, all future test utilities will fit in here growing it to potentially large size that will be hard to support.

How about `generate_test_objects`? Feel free to choose any other more specific name.

Unrelated - isn't our testing framework going to try to execute tests in this file if we are going to keep the `test_` prefix in its name?

Reply:
Bad naming seems to be a theme here :)  I like the 'generate_test_objects' suggestion.   This can apply elsewhere also, so I'll update.

This file is ignored by pytest because we are omitting the `if __name__ == __main__` block to run the file as a test.

Reply:
Thank you for the explanation about `pytest`.